pkg/utils/log.go

```diff
@@ -51,10 +51,13 @@ func logWithLongFile(l *log.Logger, format string, v ...interface{}) {
 	// From
 	// /builddir/build/BUILD/heketi-3f4a5b1b6edff87232e8b24533c53b4151ebd9c7/src/github.com/heketi/heketi/apps/glusterfs/volume_entry.go
 	// to
-	// src/github.com/heketi/heketi/apps/glusterfs/volume_entry.go
-	i := strings.Index(file, "/src/")
+	// apps/glusterfs/volume_entry.go
+	const basePath = "github.com/heketi/"
```

Comment:
Does this need to deal with paths outside of "github.com/heketi"? Or is that just not an issue for this code?

Reply:
Are you aware of any place outside of the heketi codebase, where pkg/utils/log is being used? that would surprise me. And even if, it would just not shorten the path end print the full path. I was just annoyed by the fact that I hade lot's of useless src/github.com/heketi/heketi/ in my terminal window.

Reply:
I'm +1 to making the heketi paths shorter. I don't think this version of the heketi utils logger will be used outside of heketi code, so we're probably safe. I just wanted to see if you had thought about the issue. The slightly longer story is that there's a copy of heketi utils in its own standalone repo that could in fact be used by other stuff... but since I started working on heketi we've never tried to keep that code in sync, so I am not really worried about it.

Reply:
IMO, the path had src/github.com/heketi/heketi because it is possible that heketi is built from a different Source control repo. That possibility is small, I think this is fine.